docs/secure.md

```diff
@@ -8,99 +8,72 @@
+[cert-manager](cert-manager.io), a certificate management tool for 
+Kubernetes.
+
 ### Prerequisites
 
-Before you secure MongoDBCommunity resource connections using TLS, you must:
+Before you secure MongoDBCommunity resource connections using TLS, you 
+must:
 
-1. Create a PEM-encoded TLS certificate for the servers in the MongoDBCommunity resource using your own Certificate Authority (CA). The certificate must have one of the following:
+1. [Create a database user](docs/users) to authenticate to your 
+   MongoDBCommunity resource.
+1. [Deploy a MongoDBCommunity resource](docs/deploy-configure).
 
-   - A wildcard `Common Name` that matches the domain name of all of the replica set members:
+### Procedure
 
-     ```
-     *.<metadata.name of the MongoDBCommunity resource>-svc.<namespace>.svc.cluster.local
-     ```
-   - The domain name for each of the replica set members as `Subject Alternative Names` (SAN):
+To secure connections to MongoDBCommunity resources with TLS using `cert-manager`:
 
-     ```
-     <metadata.name of the MongoDBCommunity resource>-0.<metadata.name of the MongoDBCommunity resource>-svc.<namespace>.svc.cluster.local
-     <metadata.name of the MongoDBCommunity resource>-1.<metadata.name of the MongoDBCommunity resource>-svc.<namespace>.svc.cluster.local
-     <metadata.name of the MongoDBCommunity resource>-2.<metadata.name of the MongoDBCommunity resource>-svc.<namespace>.svc.cluster.local
-     ```
+1. Add the `cert-manager` repository to your `helm` repository list and
+   ensure it's up to date:
 
-1. Create a Kubernetes ConfigMap that contains the certificate for the CA that signed your server certificate. The key in the ConfigMap that references the certificate must be named `ca.crt`. Kubernetes configures this automatically if the certificate file is named `ca.crt`:
    ```
-   kubectl create configmap <tls-ca-configmap-name> --from-file=ca.crt --namespace <namespace>
+   helm repo add jetstack https://charts.jetstack.io
+   helm repo update
    ```
 
-   For a certificate file with any other name, you must define the `ca.crt` key manually:
-   ```
-   kubectl create configmap <tls-ca-configmap-name> --from-file=ca.crt=<certificate-file-name>.crt --namespace <namespace>
-   ```
+1. Install `cert-manager`:
 
-1. Create a Kubernetes secret that contains the server certificate and key for the members of your replica set. For a server certificate named `server.crt` and key named `server.key`:
    ```
-   kubectl create secret tls <tls-secret-name> --cert=server.crt --key=server.key --namespace <namespace>
+   helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --set installCRDs=true
    ```
 
-### Procedure
-
-To secure connections to MongoDBCommunity resources using TLS:
-
-1. Add the following fields to the MongoDBCommunity resource definition:
+1. Perform an upgrade to the MongoDBCommunity resource you wish to 
```

Comment:
This would rather be `Create a TLS secured MongoDBCommunity` resource, since there's none created before this step.